.coveralls.yml

```diff
@@ -0,0 +1,2 @@
+service_name: travis-pro
+repo_token: L1JneQSHVx018toJUSN0ZqXpWx1G2QFel
```

Comment:
It's the unique ID of the [corresponding repository on Coveralls](https://coveralls.io/github/quattor/pan).